app.rb

```diff
@@ -14,19 +16,38 @@ def initialize
   end
 
   def list_all_books
-    puts 'list all books'
```

Comment:
- [OPTIONAL ] I think it would be better to specify a message when the books_list is empty like you did at lin `43`